x-pack/plugins/observability_ai_assistant/server/routes/chat/route.ts

```diff
@@ -28,7 +27,7 @@ const chatRoute = createObservabilityAIAssistantServerRoute({
       ),
     }),
   }),
-  handler: async (resources): Promise<IncomingMessage> => {
+  handler: async (resources) => {
```

Comment:
I prefer to keep this is possible, we can also change it later. The reason is that without an explicit type annotation, TypeScript will analyze the code to infer the return type. This happens every time the `callApi` function is used as it is one big object/type.

Reply:
Yes, this change was a byproduct of the new 'borrow' typing of the interface.